notebooks/1-welcome.ipynb

```diff
@@ -13,24 +13,102 @@
+    "- **an *optional module* uses an example file with Python scripts**\n",
+    "  - Learners who use other programming language can replace Python script with short script or code snippets in programming language of their preference"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "👍 **Prerequisite:**\n",
-    "- prior knowledge of using GitHub for project repositories. \n",
-    "- Basic experience working with Markdown files and Jupyter notebooks will be a plus. \n",
-    "- One **optional module** shows example using Python scripts\n",
+    "👋 Who we are\n",
+    "---\n",
+    "\n",
+    "**Instructors**\n",
+    "\n",
+    "- **Martina G. Vilas**: Core maintainer of _The Turing Way_, Open Source contributor of Jupyter Book, PhD student at Max Plank Institute, Germany. *Twitter: [@martinagvilas](https://twitter.com/martinagvilas)*\n",
+    "- **Sarah Gibson**: Core maintainer of _The Turing Way_ and Binder, contributor of Project Jupyter, Research Software Engineer, The Alan Turing Institute, UK. *Twitter: [@drsarahlgibson](https://twitter.com/drsarahlgibson)*\n",
+    "- **Malvika Sharan**: Community Manager - _The Turing Way_, Research Associate, Whitaker Lab, The Alan Turing Institute, UK. *Twitter: [@malvikasharan](https://twitter.com/malviaksharan)*\n",
+    "\n",
+    "**Project leads**\n",
+    "\n",
+    "- **Kirstie Whitaker**: Project lead - _The Turing Way_, head of the Tools, Practices, and Systems research programme, The Alan Turing Institute, UK. *Twitter: [@kirstie_j](https://twitter.com/kirstie_j)*\n",
+    "- **Chris Holdgraf**: Project lead - Jupyter Book, member of Project Jupyter and Binder, fellow at the Berkeley Institute for Data Science, USA. *Twitter: [@choldgraf](https://twitter.com/choldgraf)*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "👍 Tutorial Setup\n",
+    "---\n",
+    "\n",
+    "*This setup guide will take you from reading this page online to reading it locally.*\n",
+    "\n",
+    "**Jupyter Notebooks**\n",
+    "\n",
+    "This tutorial will make use of Jupyter Notebooks to guide the lessons (divided into short modules). \n",
+    "Please [install Anaconda](https://www.anaconda.com/), a Python distribution that comes with everything we need for the lesson. \n",
+    "Detailed installation instructions for various operating systems can be found in [Anaconda documentation](https://docs.anaconda.com/anaconda/install).\n",
+    "\n",
+    "**Jupyter Book**\n",
+    "\n",
+    "This first part of this tutorial uses Jupyter Book on personal computer. Please install the Jupyter-Book Comman Line Interface (CLI) that will allow you to build and control your Jupyter Book. \n",
+    "Please install it via pip (or pip3) using the following command:\n",
+    "\n",
+    "`pip install -U jupyter-book`\n",
```

Comment:
Alternatively, we can put this step after `git clone` and get them to run `pip install -r requirements.txt`

Reply:
I think one module explains what `requirements.txt` is. That's why I kept it simple here. what do you think?

Reply:
It depends what the purpose of this section is. If it's to get everyone set up with all the tools they need, then they should run `pip install -r requirements.txt` since jupyter book is not our only requirement.

Reply:
Sounds good. I will open a PR and add this (pretty much exactly what you just said).